### PR TITLE
C2C-37: Cleaned up antenatal related concepts

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

**Comment by reagan-meant** [comment]
@Need proof reading some of the concepts in french...

File: configuration/concepts/Consultation_prenatale.csv [hunk 1]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
-ced9398c-164a-4384-8a2f-1dba0b393669,,,Yes,Oui,Yes,Oui,,,Misc,N/A,,,,,,,,,,,
-db8e455a-dffe-4ce6-aa45-979900078ff0,,,No,Non,No,Non,,,Misc,N/A,,,,,,,,,,,
 30a99cd4-90d6-43f3-9d56-47a7f862a73c,,,Examination not done,Examen non fait,Examination not done,Examen non fait,,,Misc,N/A,,,,,,,,,,,
-3d7c6bd3-9818-4fce-8185-eb6dd38bc947,,,HIV,VIH,HIV,VIH,,,Misc,Coded,,Yes;No;Examination not done,,,,,,,,,
-ced9398c-164a-4384-8a2f-1dba0b393669,,,Yes,Oui,Yes,Oui,,,Misc,N/A,,,,,,,,,,,
-db8e455a-dffe-4ce6-aa45-979900078ff0,,,No,Non,No,Non,,,Misc,N/A,,,,,,,,,,,
-30a99cd4-90d6-43f3-9d56-47a7f862a73c,,,Examination not done,Examen non fait,Examination not done,Examen non fait,,,Misc,N/A,,,,,,,,,,,
-386b9d40-266e-47f1-ac32-d46f89dfe90b,,,Syphilis prenatal,Syphilis prenatal,Syphilis,Syphilis,,,Misc,Coded,,Yes;No;Examination not done,,,,,,,,,
 f258fbef-d1e5-47db-b3dd-d31b0a438a76,,,Risk pregnancy,Grossesse à risque,Risk pregnancy,Grossesse à risque,,,Misc,Boolean,,,,,,,,,,,
 e28bf540-37ab-4ca2-9197-5e94f75668a7,,,"If yes, Multiple pregnancy","Si oui, Grossesse multiple","If yes, Multiple pregnancy","Si oui, Grossesse multiple",,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Consultation_prenatale.csv:41]: Have we already answered this on line 29?

File: configuration/concepts/Consultation_prenatale.csv [hunk 2]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
+4550dd56-ee8f-4f47-beb4-50ffc448715f,,CIEL:47,Pregnancy + hypertension,Grossesse + hypertension,Pregnancy + hypertension,Grossesse + hypertension,,,Misc,N/A,,,,,,,,,,,
 dc759555-9e47-4ddc-8dc4-1f10024e7248,,,Pregnancy + HIV / Syphilis,Grossesse + VIH/Syphilis,Pregnancy + HIV / Syphilis,Grossesse + VIH/Syphilis,,,Misc,N/A,,,,,,,,,,,
 815239ab-6321-4d48-86d4-1e6d8f47a94b,,,Pregnancy + obesity,Grossesse + obésité,Pregnancy + obesity,Grossesse + obésité,,,Misc,N/A,,,,,,,,,,,
-9ea362b1-72a2-4b5e-9bde-13572f37de56,,,Pregnancy + malnutrition,Grossesse + malnutrition,Pregnancy + malnutrition,Grossesse + malnutrition,,,Misc,N/A,,,,,,,,,,,
+9ea362b1-72a2-4b5e-9bde-13572f37de56,,CIEL:157648,Pregnancy + malnutrition,Grossesse + malnutrition,Pregnancy + malnutrition,Grossesse + malnutrition,,,Misc,N/A,,,,,,,,,,,
 50e4eb00-968a-456f-824b-cbe320064dcf,,,Pregnancy below 18 years of age / above 36 years of age,Grossesse en bas de 18 ans / en haut de 36 ans,Pregnancy below 18 years of age / above 36 years of age,Grossesse en bas de 18 ans / en haut de 36 ans,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Consultation_prenatale.csv:47]: Should we rename to: "Mother below 18 years of age / above 36 years of age"
Or even "Mother younger than 18 or older that 36".
In French:
'Mère âgée de moins de 18 ans ou de plus de 36 ans'

File: configuration/concepts/Consultation_prenatale.csv [hunk 3]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
-96129acc-86f4-409b-7bf7-db0934278b4a,,,Prenatal consultation form,Formulaire de Consultation prenatale,Prenatal consultation form,Formulaire de Consultation prenatale,,,ConvSet,N/A,,,"Visit;First pregnancy;Number of previous pregnancies;Childbirth during the last pregnancy;Places of delivery of the last pregnancy;If other, where;Date of most recent pregnancy delivery;Complications during the most recent delivery;If yes describe the complications;Number of living children;Date of last period;Expected delivery date;Pregnancy age in weeks;Uterine height;Presentation;Fetal heart rate;Multiple pregnancy;Symptoms;If other;Physical examination notes;HIV;Syphilis prenatal;Risk pregnancy;If yes, why;Childbirth plan;Where the patient plans to give birth",,,,,,,,
+6ef80ed0-2102-45d3-a188-ad57c16263dc,,,Where the patient plans to give birth,Où la patiente prévoit elle accoucher?,Where the patient plans to give birth,Où la patiente prévoit elle accoucher?,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritan Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;Borgne Hospital;Trou du Nord Hospital;Home;Other,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Consultation_prenatale.csv:50]: "Où la patiente prévoit-elle d'accoucher?"
That sounds more like a direct question, ie, the label than a concept really.
Should we set something like 'Intended place/location of the delivery' (exact term yet to be found - feel free to improve it) and rather have the label in the form to be "Where the patient plans to give birth?"
In French that'd be "Lieu prévu de l'accouchement"

File: configuration/concepts/Consultation_prenatale.csv [hunk 4]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
-1ff3db17-7a6f-40bf-8062-a912773e905c,,,Good Samaritan Limbe Hospital,Hopital Bon Samaritain Limbe,Good Samaritan Limbe Hospital,Hopital Bon Samaritain Limbe,,,Misc,N/A,,,,,,,,,,,
-b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Justinien Cap Haitien,Justinien Cap Haitien Hospital,Hopital Justinien Cap Haitien,,,Misc,N/A,,,,,,,,,,,
-696114ce-8c7b-4018-a8d5-ee2e65d323d3,,,Sacred Heart Milot Hospital,Hopital Sacré Coeur Milot,Sacred Heart Milot Hospital,Hopital Sacré Coeur Milot,,,Misc,N/A,,,,,,,,,,,
-50f4de56-b5b6-4939-aa73-7d7f5a5f8961,,,HBCH Hospital,Hopital HBCH,HBCH Hospital,Hopital HBCH,,,Misc,N/A,,,,,,,,,,,
-57195bab-5935-4da4-b5de-e9ee1dba162e,,,Gable Hospital,Hopital de Pignon,Gable Hospital,Hopital de Pignon,,,Misc,N/A,,,,,,,,,,,
-1c61317b-f7cf-4709-829c-76abb6f6501b,,,One-eyed Hospital,Hopital de Borgne,One-eyed Hospital,Hopital de Borgne,,,Misc,N/A,,,,,,,,,,,
-6d1450bb-0109-456b-82f9-e495bcb5c959,,,Trou du Nord Hospital,Hopital de Trou du Nord,Trou du Nord Hospital,Hopital de Trou du Nord,,,Misc,N/A,,,,,,,,,,,
-df9cc02b-b427-4262-b30a-7b973aef23fc,,,At home,À la maison,At home,À la maison,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Consultation_prenatale.csv:78]: "A domicile"

**Review by rbuisson — CHANGES_REQUESTED** [review]
Made quite some comments on the French translations and few on the English naming too.
Let me know if you want need to discuss it over a call.

File: configuration/concepts/Consultation_prenatale.csv [hunk 5]
```diff
@@ -1,16 +1,12 @@
...
-5bab781c-96be-4c32-b75b-d7906ddff555,,,First pregnancy,Première grossesse,First pregnancy,Première grossesse,,,Misc,Boolean,,,,,,,,,,,
-e24e688d-9708-4409-bbef-d4b148392926,,,Number of previous pregnancies,Nombre de grossesses précédentes,Number of previous pregnancies,Nombre de grossesses précédentes,,,Misc,Numeric,,,,,,,,,,,
-dfb79477-ab61-47c4-b308-e1ab09757c18,,,Prenatal - vaginal consultation,Consultation prenatale - vaginal,Prenatal - vaginal consultation,Consultation prenatale - vaginal,,,Misc,N/A,,,,,,,,,,,
-5066f452-5d8b-4e04-b815-0c212929e47e,,,Cesarean,Césarienne,Cesarean,Césarienne,,,Misc,N/A,,,,,,,,,,,
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1200
+c4b48a7a-7c13-4e7d-89ad-1df68c5c6801,,CIEL:1425,Visit Number,Nombre de visite,Visit number,Nombre de visite,,,Finding,Numeric,,,,FALSE,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 6]
```diff
@@ -1,16 +1,12 @@
...
+5bab781c-96be-4c32-b75b-d7906ddff555,,,Pregnancy number,Nombre de grossesse,Pregnancy number,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
+b346c76b-56ba-4c27-97cc-f7b0f85bf581,,CIEL:160080,Number of Term Pregnancies,Nombre de grossesse,Number of Term Pregnancies,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
+1b6a7d76-755c-4db9-8f05-42fe28fc3175,,CIEL:160601,Number of live births,Nombre de grossesse,Number of live births,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 7]
```diff
@@ -1,16 +1,12 @@
...
-e24e688d-9708-4409-bbef-d4b148392926,,,Number of previous pregnancies,Nombre de grossesses précédentes,Number of previous pregnancies,Nombre de grossesses précédentes,,,Misc,Numeric,,,,,,,,,,,
-dfb79477-ab61-47c4-b308-e1ab09757c18,,,Prenatal - vaginal consultation,Consultation prenatale - vaginal,Prenatal - vaginal consultation,Consultation prenatale - vaginal,,,Misc,N/A,,,,,,,,,,,
-5066f452-5d8b-4e04-b815-0c212929e47e,,,Cesarean,Césarienne,Cesarean,Césarienne,,,Misc,N/A,,,,,,,,,,,
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1200
+c4b48a7a-7c13-4e7d-89ad-1df68c5c6801,,CIEL:1425,Visit Number,Nombre de visite,Visit number,Nombre de visite,,,Finding,Numeric,,,,FALSE,,,,,,,
+5bab781c-96be-4c32-b75b-d7906ddff555,,,Pregnancy number,Nombre de grossesse,Pregnancy number,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 8]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
+4ba98e11-cb3b-4a22-be4e-bb43f3078683,,CIEL:163750,Uterine contractions,Contractions utérine,Uterine contractions,Contractions utérine,,,Misc,N/A,,,,,,,,,,,
+7f703275-777c-4237-9d07-36c99ea0966c,,CIEL:153551,Abdominal pain,Douleur abdominale,Abdominal pain,Douleur abdominale,,,Misc,N/A,,,,,,,,,,,
 6f8988d1-157a-44cc-ac3b-22b627a57db3,,,Fever,Fièvre,Fever,Fièvre,,,Misc,N/A,,,,,,,,,,,
-e90fa130-09cd-4574-936f-91fca52a9324,,,Swollen feet,Pieds enflés,Swollen feet,Pieds enflés,,,Misc,N/A,,,,,,,,,,,
-8d83ff2b-3f6f-432a-b0cd-5b4f2f1055fd,,,Vision problems,Troubles de vision,Vision problems,Troubles de vision,,,Misc,N/A,,,,,,,,,,,
-26c0366b-83e4-4401-9699-e545219b8870,,,Other,Autre,Other,Autre,,,Misc,N/A,,,,,,,,,,,
-f7312165-7e96-4761-b4cf-d601a1a18f9c,,,Symptoms,Symptômes,Symptoms,Symptômes,,,Misc,Coded,,Vaginal bleeding;Uterine contractions;Abdominal bread;Fever;Swollen feet;Vision problems;Other,,,,,,,,,
+e90fa130-09cd-4574-936f-91fca52a9324,,CIEL:125198,Swollen feet,Pieds enflés,Swollen feet,Pieds enflés,,,Misc,N/A,,,,,,,,,,,
+8d83ff2b-3f6f-432a-b0cd-5b4f2f1055fd,,CIEL:118939,Vision problems,Troubles de vision,Vision problems,Troubles de vision,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv:41]: It appears the intention is to be able to capture the reason as to why the radiographer thinks it is a risky pregnancy

File: configuration/concepts/Consultation_prenatale.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: "Mother younger than 18 or older than 36 years" sounds better...
Done

File: configuration/concepts/Consultation_prenatale.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Added "Planned place/location for delivery".....confirm the fr..."eu / lieu prévu pour la livraison"

File: configuration/concepts/Consultation_prenatale.csv [hunk 9]
```diff
@@ -1,16 +1,12 @@
...
-5066f452-5d8b-4e04-b815-0c212929e47e,,,Cesarean,Césarienne,Cesarean,Césarienne,,,Misc,N/A,,,,,,,,,,,
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1200
+c4b48a7a-7c13-4e7d-89ad-1df68c5c6801,,CIEL:1425,Visit Number,Nombre de visite,Visit number,Nombre de visite,,,Finding,Numeric,,,,FALSE,,,,,,,
+5bab781c-96be-4c32-b75b-d7906ddff555,,,Pregnancy number,Nombre de grossesse,Pregnancy number,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
+b346c76b-56ba-4c27-97cc-f7b0f85bf581,,CIEL:160080,Number of Term Pregnancies,Nombre de grossesse,Number of Term Pregnancies,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
+1b6a7d76-755c-4db9-8f05-42fe28fc3175,,CIEL:160601,Number of live births,Nombre de grossesse,Number of live births,Nombre de grossesse,,,Misc,Numeric,,,,FALSE,,,,,,,
+dfb79477-ab61-47c4-b308-e1ab09757c18,,CIEL:162595,Vaginal delivery,Vaginal delivery,Vaginal delivery,Vaginal delivery,,,Finding,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: done

File: configuration/concepts/Consultation_prenatale.csv [hunk 10]
```diff
@@ -1,16 +1,12 @@
...
+5066f452-5d8b-4e04-b815-0c212929e47e,,CIEL:1171,Cesarean,Césarienne,Cesarean,Césarienne,,,Misc,N/A,,,,,,,,,,,
 90950144-af82-460d-9393-e5e412445075,,,No workbirth (Abortion),Aucun accouchement (Avortement),No workbirth (Abortion),Aucun accouchement (Avortement),,,Misc,N/A,,,,,,,,,,,
-2a07d3b8-c3d5-4fae-ac89-e0a1dfdd9915,,,Childbirth during the last pregnancy,Accochement lors de la dernière grossesse,Childbirth during the last pregnancy,Accochement lors de la dernière grossesse,,,Misc,Coded,,Prenatal - vaginal consultation;Cesarean;No workbirth (Abortion),,,,,,,,,
+2a07d3b8-c3d5-4fae-ac89-e0a1dfdd9915,,CIEL:161033,Birth outcome during the last pregnancy,Accochement lors de la dernière grossesse,Birth outcome during the last pregnancy,Accochement lors de la dernière grossesse,,,Misc,Coded,,Vaginal delivery;Cesarean;No workbirth (Abortion),,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 11]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
 57195bab-5935-4da4-b5de-e9ee1dba162e,,,Gable Hospital,Hopital de Pignon,Gable Hospital,Hopital de Pignon,,,Misc,N/A,,,,,,,,,,,
 1c61317b-f7cf-4709-829c-76abb6f6501b,,,Borgne Hospital,Hopital de Borgne,Borgne Hospital,Hopital de Borgne,,,Misc,N/A,,,,,,,,,,,
 6d1450bb-0109-456b-82f9-e495bcb5c959,,,Trou du Nord Hospital,Hopital de Trou du Nord,Trou du Nord Hospital,Hopital de Trou du Nord,,,Misc,N/A,,,,,,,,,,,
-df9cc02b-b427-4262-b30a-7b973aef23fc,,,At home,À la maison,At home,À la maison,,,Misc,N/A,,,,,,,,,,,
-26c0366b-83e4-4401-9699-e545219b8870,,,Other,Autre,Other,Autre,,,Misc,N/A,,,,,,,,,,,
-023e0a9e-cf47-4a4d-9cf4-b2edfbe7fc33,,,Places of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,Places of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritan Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;One-eyed Hospital;Trou du Nord Hospital;At home;Other,,,,,,,,,
+df9cc02b-b427-4262-b30a-7b973aef23fc,,CIEL:1536,Home,Maison,Home,Maison,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 12]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
+023e0a9e-cf47-4a4d-9cf4-b2edfbe7fc33,,CIEL:1572,Places of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,Places of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritan Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;Borgne Hospital;Trou du Nord Hospital;Home;Other,,,,,,,,,
 8c911781-7a2b-4e8e-9ece-ad6c40928aac,,,"If other, where","Si autre, où?","If other, where","Si autre, où?",,,Misc,Text,,,,,,,,,,,
-50a37eff-81a3-4cef-a19c-1f85d4e690d6,,,Date of most recent pregnancy delivery,Date de l'accouchement de la grossesse la plus récent,Date of most recent pregnancy delivery,Date de l'accouchement de la grossesse la plus récent,,,Misc,Date,,,,,,,,,,,
+50a37eff-81a3-4cef-a19c-1f85d4e690d6,,CIEL:1596,Date of most recent pregnancy delivery,Date de l'accouchement de la grossesse la plus récent,Date of most recent pregnancy delivery,Date de l'accouchement de la grossesse la plus récent,,,Misc,Date,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 13]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
 e3892a91-1428-4816-8e83-834dcd738723,,,Complications during the most recent delivery,Complications lors de l'accouchement le plus récent ?,Complications during the most recent delivery,Complications lors de l'accouchement le plus récent ?,,,Misc,Boolean,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 14]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
-2c1e2efe-74cb-475d-a922-3785e02e7bd3,,,Vaginal bleeding,Saignements vaginaux,Vaginal bleeding,Saignements vaginaux,,,Misc,N/A,,,,,,,,,,,
-4ba98e11-cb3b-4a22-be4e-bb43f3078683,,,Uterine contractions,Contractions utérine,Uterine contractions,Contractions utérine,,,Misc,N/A,,,,,,,,,,,
-7f703275-777c-4237-9d07-36c99ea0966c,,,Abdominal bread,Douleur abdominale,Abdominal bread,Douleur abdominale,,,Misc,N/A,,,,,,,,,,,
+3b7d4768-8fc6-4a2d-9a83-05bd288b7108,,CIEL:1825,Number of living children,Nombre d'enfants vivants,Number of living children,Nombre d'enfants vivants,,,Misc,Numeric,,,,,,,,,,,
+6ed50023-d807-492f-a203-8ce87a942a52,,CIEL:1427,Date of last period,Date des  dernières règles,Date of last period,Date des  dernières règles,,,Misc,Date,,,,,,,,,,,
+6422a83f-acc6-4953-8239-f18a0b6d90a4,,CIEL:5596,Expected delivery date,Date prévue d'accouchement,Expected delivery date,Date prévue d'accouchement,,,Misc,Date,,,,,,,,,,,
+7bfa7b15-f519-4d5f-b53b-3a408bfbeedb,,CIEL:1789,Pregnancy age in weeks,Âge de la grossesse en semaines,Pregnancy age in weeks,Âge de la grossesse en semaines,,,Misc,Numeric,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 15]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
-1ba7f1d1-9644-4d5c-99af-914d1efe3d8a,,,Presentation,Présentation,Presentation,Présentation,,,Misc,Coded,,Cephalic;Seat;Transverse;Variable,,,,,,,,,
-7f10123f-bffd-47da-b8ce-6e82b25afbcc,,,Fetal heart rate,Rythme cardiaqe du fétus,Fetal heart rate,Rythme cardiaqe du fétus,,,Misc,Text,,,,,,,,,,,
-ec5216e0-aa54-41e5-8f8a-440295734cb4,,,Multiple pregnancy,Grossesse multiple,Multiple pregnancy,Grossesse multiple,,,Misc,Boolean,,,,,,,,,,,
-2c1e2efe-74cb-475d-a922-3785e02e7bd3,,,Vaginal bleeding,Saignements vaginaux,Vaginal bleeding,Saignements vaginaux,,,Misc,N/A,,,,,,,,,,,
-4ba98e11-cb3b-4a22-be4e-bb43f3078683,,,Uterine contractions,Contractions utérine,Uterine contractions,Contractions utérine,,,Misc,N/A,,,,,,,,,,,
-7f703275-777c-4237-9d07-36c99ea0966c,,,Abdominal bread,Douleur abdominale,Abdominal bread,Douleur abdominale,,,Misc,N/A,,,,,,,,,,,
+3b7d4768-8fc6-4a2d-9a83-05bd288b7108,,CIEL:1825,Number of living children,Nombre d'enfants vivants,Number of living children,Nombre d'enfants vivants,,,Misc,Numeric,,,,,,,,,,,
+6ed50023-d807-492f-a203-8ce87a942a52,,CIEL:1427,Date of last period,Date des  dernières règles,Date of last period,Date des  dernières règles,,,Misc,Date,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 16]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
+6422a83f-acc6-4953-8239-f18a0b6d90a4,,CIEL:5596,Expected delivery date,Date prévue d'accouchement,Expected delivery date,Date prévue d'accouchement,,,Misc,Date,,,,,,,,,,,
+7bfa7b15-f519-4d5f-b53b-3a408bfbeedb,,CIEL:1789,Pregnancy age in weeks,Âge de la grossesse en semaines,Pregnancy age in weeks,Âge de la grossesse en semaines,,,Misc,Numeric,,,,,,,,,,,
+f857c495-98d2-4097-9018-d6641a8304a5,,CIEL:1439,Fundal height,Hauteur utérine,Fundal height,Hauteur utérine,,,Misc,Text,,,,,,,,,,,
+ec5216e0-aa54-41e5-8f8a-440295734cb4,,CIEL:115491,Multiple pregnancy,Grossesse multiple,Multiple pregnancy,Grossesse multiple,,,Misc,Boolean,,,,,,,,,,,
+2c1e2efe-74cb-475d-a922-3785e02e7bd3,,CIEL:147232,Vaginal bleeding,Saignements vaginaux,Vaginal bleeding,Saignements vaginaux,,,Misc,N/A,,,,,,,,,,,
+4ba98e11-cb3b-4a22-be4e-bb43f3078683,,CIEL:163750,Uterine contractions,Contractions utérine,Uterine contractions,Contractions utérine,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 17]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
+7f703275-777c-4237-9d07-36c99ea0966c,,CIEL:153551,Abdominal pain,Douleur abdominale,Abdominal pain,Douleur abdominale,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

File: configuration/concepts/Consultation_prenatale.csv [hunk 18]
```diff
@@ -19,63 +15,36 @@ b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Ju
...
 6f8988d1-157a-44cc-ac3b-22b627a57db3,,,Fever,Fièvre,Fever,Fièvre,,,Misc,N/A,,,,,,,,,,,
-e90fa130-09cd-4574-936f-91fca52a9324,,,Swollen feet,Pieds enflés,Swollen feet,Pieds enflés,,,Misc,N/A,,,,,,,,,,,
-8d83ff2b-3f6f-432a-b0cd-5b4f2f1055fd,,,Vision problems,Troubles de vision,Vision problems,Troubles de vision,,,Misc,N/A,,,,,,,,,,,
-26c0366b-83e4-4401-9699-e545219b8870,,,Other,Autre,Other,Autre,,,Misc,N/A,,,,,,,,,,,
-f7312165-7e96-4761-b4cf-d601a1a18f9c,,,Symptoms,Symptômes,Symptoms,Symptômes,,,Misc,Coded,,Vaginal bleeding;Uterine contractions;Abdominal bread;Fever;Swollen feet;Vision problems;Other,,,,,,,,,
+e90fa130-09cd-4574-936f-91fca52a9324,,CIEL:125198,Swollen feet,Pieds enflés,Swollen feet,Pieds enflés,,,Misc,N/A,,,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (reagan-meant) [reply configuration/concepts/Consultation_prenatale.csv]: Done

**Pull request closed without merging**